nntrainer/layers/conv2d_layer.cpp

```diff
@@ -81,8 +78,8 @@ static void col2im(const Tensor &col_matrix, const TensorDim &kdim,
   unsigned im_eff_width = im_width + pl + pr;
   image.setZero();
 
-  int h_stride_end = im_eff_height - eff_k_height - pb;
-  int w_stride_end = im_eff_width - eff_k_width - pr;
+  int h_stride_end = im_eff_height - eff_k_height - pt;
```

Comment:
can you make this change for pooling2d layer as well.

Reply:
Right... Updated :)